tests/drivers/disk/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  drivers.disk.usdhc:
```

Comment:
I guess `platform_allow` can be replaced by `filter: CONFIG_SDMMC_USDHC`

with https://github.com/zephyrproject-rtos/zephyr/pull/40611, you can add a test case for any platform:
```
  drivers.disk.ram:
    min_ram: 128
    tags: disk
    extra_configs:
        - CONFIG_DISK_DRIVER_RAM=y
...
```

Reply:
Added this test case. I'm assuming #40611 is required to make this test pass, since it checks for out of bounds r/w?

Reply:
thanks for the heads up, updated the testcase.yaml file to use `filter: CONFIG_SDMMC_USDHC`